db/migrate/20210311183452_add_image_url_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddImageUrlToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :image_url, :string, null: true
```

Comment:
Here it's kind of pointless to have two migrations. You can add both columns in the same file.